docker.dev.yml

```diff
@@ -1,10 +1,22 @@
 version: "3"
 
 services:
-  app:
-    container_name: scribble-ocr
+  train:
+    container_name: scribble-ocr-train
     build:
       context: .
+      target: tesseractbuild
+      dockerfile: Dockerfile_tesseract_training
+    ports:
+      - 8000:8000
+    volumes:
+      - "./data:/train/tesstrain/data"
+    command: sh /app/start.sh
+  api:
+    container_name: scribble-ocr-api
```

Comment:
We also need an environment variable for `GOOGLE_CREDS` because the way the code is written, if a valid credential JSON isn't passed to that environment variable then the API doesn't actually run.